crates/iroha_cli/README.md

```diff
@@ -109,11 +68,11 @@ You can also filter based on either account, asset or domain id by using the fil
 Here are some examples of filtering:
 
 ```bash
-# Filter domains by id
+# FIXME Filter domains by id
 ./iroha domain list filter '{"Identifiable": {"Is": "wonderland"}}'
-# Filter accounts by domain
+# FIXME Filter accounts by domain
 ./iroha account list filter '{"Identifiable": {"EndsWith": "@wonderland"}}'
-# Filter asset by domain
+# FIXME Filter asset by domain
 ./iroha asset list filter '{"Or": [{"Identifiable": {"Contains": "#wonderland#"}}, {"And": [{"Identifiable": {"Contains": "##"}}, {"Identifiable": {"EndsWith": "@wonderland"}}]}]}'
```

Comment:
FIXME query filter examples. For now post-process like `jq` would work

Reply:
#5294

Reply:
If `iroha * list filter` has an implicit projection to ids, post-processes cannot filter by other columns